docs/site/todo-list-tutorial-model.md

```diff
@@ -0,0 +1,115 @@
+---
+lang: en
+title: 'Add TodoList Model'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-model.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Model
+---
+
```

Comment:
nitpick: how about link to it's previous tutorial at the beginning?

Reply:
done